bucket/janet.json

```diff
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://janet-lang.org",
+    "description": "Janet is a lisp-like, embeddable, functional and imperative programming language and bytecode interpreter.",
+    "license": "MIT",
+    "version": "1.13.1",
+    "architecture": {
+        "32bit": {
+            "hash": "627a0dccef7815fdde735caf3ea2eb3e16f2ec26ac3d0cc365cf8ea67589d5e3",
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x86-installer.msi"
+        },
+        "64bit": {
+            "hash": "ef5d32257594c8ab54aaa52fe8e84bb951a4883bbb6f6848e845759122d33a35",
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x64-installer.msi"
+        }
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://github.com/janet-lang/janet/releases/download/v$version/janet-v$version-windows-x86-installer.msi"
+            },
+            "64bit": {
+                "url": "https://github.com/janet-lang/janet/releases/download/v$version/janet-v$version-windows-x64-installer.msi"
+            }
+        }
+    },
+    "checkver": {
+        "github": "https://github.com/janet-lang/janet"
+    },
+    "bin": [
+        "bin\\janet.exe",
+        ["bin\\janet.exe", "jpm", "$dir\\bin\\jpm"]

```

Comment:
- format using checkver/formatjson
- What is the parameter for? Documentation link?

Reply:
`formatjson`: Thanks. Great tip!

I replaced this shim with the one to use jpm.bat directly, similar to how stuff works in `psutils`. Also I removed `PATH` pollution as this is the major advantage of scoop and `janet` doesn't seem to care once other environment variables are set properly.

`jpm` is a package manager written in janet. It's a janet cousin of `npm`, `yarn` or `cargo`.

Reply:
I've found a link to the documentation: https://janet-lang.org/docs/jpm.html